skymap_scanner/utils/extract_json_message.py

```diff
@@ -100,9 +102,9 @@ def __extract_frame_packet(
     frame_packet,
     reco_algo: str,
     is_real_event: bool,
-    cache_dir="./cache/",
-    GCD_dir : str = str(cfg.DEFAULT_GCD_DIR),
-    pulsesName="SplitUncleanedInIcePulses",
+    pulsesName : str,
+    cache_dir : str,
+    GCD_dir : str,
```

Comment:
Please excuse the pedantry for a moment. For pep8-compliance, there should be no space before `:`, like `pulsesName: str` instead of `pulsesName : str`. Looks like there are several places with this.

Reply:
No worries, but how come black 23.1 is not enforcing this?

Reply:
Not even `autopep8` (2.0.2, `pycodestyle` 2.10.0) suggests changes. I took care of it manually.

Reply:
`# fmt: off` is at the top of the file. I put that there to limit needless diffs